packages/trpc/server/routers/loggedInViewer/deleteCredential.handler.ts

```diff
@@ -328,52 +327,37 @@ export const deleteCredentialHandler = async ({ ctx, input }: DeleteCredentialOp
     }
   }
 
-  // Validated that credential is user's above
-  await prisma.credential.delete({
-    where: {
-      id: id,
-    },
-  });
-
   // Backwards compatibility. Selected calendars cascade on delete when deleting a credential
   // If it's a calendar remove it from the SelectedCalendars
   if (credential.app?.categories.includes(AppCategories.calendar)) {
-    const selectedCalendars = await prisma.selectedCalendar.findMany({
-      where: {
-        userId: user.id,
-        integration: credential.type as string,
-      },
-    });
-
-    if (selectedCalendars.length) {
+    try {
```

Comment:
Wrapping in a try/catch block since if there is an error here. It shouldn't stop the deletion of the credential.